addon/serializers/echonest.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+const { isPresent } = Ember;
+const { RESTSerializer } = DS;
+
+export default RESTSerializer.extend({
+    modelKey: '',
```

Comment:
I'm setting a sane default (and will document it), but I'd entertain the idea of `null` instead of `''` here.